### PR TITLE
Learning Joulrnal Final Step

### PR DESCRIPTION
**Pull request by SakiFu**
Final step

### CODE REVIEW

File: journal.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+@view_config(route_name='edit', renderer='templates/edit.jinja2')
+def edit_view(request):
+    entries = Entry.all()
+    return {'entries':entries}
+
+@view_config(route_name='create', renderer='templates/create.jinja2')
+def edit_view(request):
+    entries = Entry.all()
+    return {'entries':entries}
```
Comment (cewing) [inline journal.py:75]: Why do you have the same view listed twice?

File: journal.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    config.include('pyramid_jinja2')
+    config.include('pyramid_tm')
+    config.add_route('home', '/')
+    config.add_route('detail', '/detail')
+    config.add_route('add', '/add')
+    config.add_route('create', '/create')
+    config.add_route('edit', '/edit')
+    config.add_route('login', '/login')
+    config.add_route('logout', '/logout')
+    config.add_route('other', '/other/{special_val')
```
Comment (cewing) [inline journal.py:154]: This route is not attached to any view.  You should remove it.

File: templates/base.jinja2 [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+
+    <!-- ADD THE FOLLOWING LINE ONLY -->
+    <link href="/static/style.css" rel="stylesheet" type="text/css">
+  </head>
+  <body>
+    <header>
+      <div id='title-1'>
+            <img src="static/nmr_2.png" class="img" width='100px' height='70px'>
```
Comment (cewing) [inline templates/base.jinja2:17]: good use of the static view to get at your (_super_ creepy) logo image :)

File: templates/detail.jinja2 [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  <div class="field">
+    <label for="text">Text</label>
+    <textarea name="text" id="text" rows="5" cols="80"></textarea>
+  </div>
+  <div class="control_row">
+    <input type="submit" value="Share" name="Share"/>
+  </div>
+</form>
+</aside>
+{% endif %}
```
Comment (cewing) [inline templates/detail.jinja2:20]: having the edit form in the detail page is a bit odd.

File: templates/edit.jinja2 [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    <div class="entry_body">
+      {{ entry.text|safe }}
+    </div>
+  </article>
+  {% else %}
+  <div class="entry">
+    <p><em>No entries here so far</em></p>
+  </div>
+  {% endfor %}
+{% endblock %}
```
Comment (cewing) [inline templates/edit.jinja2:35]: Not sure about the purpose of this template as opposed to all the others with the same content.

File: templates/base.jinja2 (hunk 3 above)
Comment (SakiFu) [inline templates/base.jinja2:17]: I think I forgot to remove unnecessary codes, which I put when I was trying
to figure things out. I'll remove them.

**Pull request merged**